erp-data-exchange-specification.md

```diff
@@ -304,15 +304,93 @@ These options are outlined here:
 | **receiveAdminNotifications** | boolean                                           | Receives Notification-Mails for Tickets with no Assignee. Default setting is true                                                                                 |
 | **reportEmails**              | Array of Strings                                  | List of email addresses which should receive report emails for this job                                                                                           |
 | **unitType**                  | string                                            | Controls the type of import Units, one of type 'rented' or 'owned'                                                                                                |
+| **webhooks**                  | Array of [webhooks](#webhooks)                    | List of [webhooks](#webhooks) that can be called for each job, triggered by defined events.                                                                       |
 
 #### Example
 
 ```json
 {
-  "autoImport": false,
-  "locale": "de_DE",
-  "receiveAdminNotifications": false,
-  "reportEmails": ["mr.foo@bar.test", "mrs.foo@bar.test"]
+	"autoImport": false,
+	"locale": "de_DE",
+	"receiveAdminNotifications": false,
+	"reportEmails": ["mr.foo@bar.test", "mrs.foo@bar.test"],
+	"webhooks": [
+		{
+			"description": "allthings#slackbot-noise",
+			"format": "slack",
+			"triggers": true,
+			"url": "https://hooks.slack.com/services/"
+		}
+	]
+}
+```
+
+### webhooks
+
+If webhooks are defined, each webhook is called when the defined event triggers. The webhooks can be defined in the import configuration or in the manifest.json.
+The webhooks from the manifest.json are merged to the configuration.  
+With this, a customer can define a webhook and also the event when it should be called.
+
+The options webhooks needed are outlined here:
+
+#### Fields
+
+| Field           | Type                        | Description                                                                                                                                        |
+| --------------- | --------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **description** | string                      | A description or name of the webhook.                                                                                                              |
+| **format**      | string                      | Either `'slack'` or `'rest'`                                                                                                                       |
```

Comment:
I wonder if it should be `post` instead of `rest` since `post` is more representative (and also what's most common in my experience with webhooks)

Reply:
I was thinking about the format to be a rest-api that is called or in case of slack, a service.